doc/source/release-notes/Percona-Server-5.6.26-74.0.rst

```diff
@@ -0,0 +1,83 @@
+
+ Querying :table:`GLOBAL_TEMPORARY_TABLES` table would crash threads working with internal temporary tables used by ``ALTER TABLE``. Bug fixed :bug:`1113388`.
+
+ Selecting from :table:`GLOBAL_TEMPORARY_TABLES` table while running an online ``ALTER TABLE`` on a partitioned table in parallel could lead to a server crash. Bug fixed :bug:`1193264`.
+
+ :ref:`innodb_kill_idle_trx` feature could cause an assertion on a debug build due to a race condition. Bug fixed :bug:`1206008`.
+
+ ``libmylsqclient_16`` symbols were missing in |Percona Server| shared library package on *RHEL*/*CentOS* 7. Bug fixed :bug:`1420691`.
+
+ Prepared statements in stored procedures could crash :ref:`response_time_distribution` plugin. Bug fixed :bug:`1426345`.
+
+ When variable :variable:`innodb_corrupt_table_action` is set to ``Warn/Salvage`` then server could crash on updating table statistics during query execution on affected tables. Bug fixed :bug:`1426610`.
+
+ A sequence of failing ``TRUNCATE TABLE``, then insert to that table, and ``CHECK TABLE`` would crash the server. Bug fixed :bug:`1433197`.
+
+ When |InnoDB| change buffering was enabled and used, executing a ``FLUSH TABLE ... FOR EXPORT`` would cause a server hang and ``SHOW PROCESSLIST`` would show that table in a ``System Lock`` state. Bug fixed :bug:`1454441` (upstream :mysqlbug:`77011`).
+
+ ``FLUSH INDEX_STATISTICS`` / ``FLUSH CHANGED_PAGE_BITMAPS`` and ``FLUSH USER_STATISTICS`` / ``RESET CHANGE_PAGE_BITMAPS`` pairs of commands were inadvertently joined, i.e. issuing either command had the effect of both. The first pair, besides flushing both index statistics and changed page bitmaps, had the effect of ``FLUSH INDEX_STATISTICS`` requiring ``SUPER`` instead of ``RELOAD`` privilege. The second pair resulted in ``FLUSH USER_STATISTICS`` destroying changed page bitmaps. Bug fixed :bug:`1472251`.
+
+ Enabling :variable:`super_read_only` together with :variable:`read_only` in :file:`my.cnf` would result in server crashing on startup. The workaround is to enable :variable:`super_read_only` dynamically on a running server.Bug fixed :bug:`1389935` ( the fix was ported from Facebook patch `#14d5d9 <https://github.com/percona/percona-server/commit/14d5d9a94f2cac1ae67afd8a806a0ed581530f7e>`_).
+
+ Enabling :variable:`super_read_only` as a command line option would not enable :variable:`read_only`. Bug fixed :bug:`1389935` ( the fix was ported from Facebook patch `#14d5d9 <https://github.com/percona/percona-server/commit/14d5d9a94f2cac1ae67afd8a806a0ed581530f7e>`_).
+
+ If a new connection thread was created while a ``SHOW PROCESSLIST`` command or a :table:`INFORMATION_SCHEMA.PROCESSLIST` query was in progress, it could have a negative TIME_MS value returned in the PROCESSLIST output. Bug fixed :bug:`1379582`.
+
+ Running ``INSERT`` queries for pessimistic B-tree inserts with :ref:`innodb_fake_changes_page` enabled could lead to server assertion with ``unknown error code 1000``. Bug fixed :bug:`1410410`.
```

Comment:
With fake changes enabled, a write to an InnoDB table that would cause B-tree reorganisation could lead to ...